cmd/buildah/inspect.go

```diff
@@ -96,7 +97,12 @@ func inspectCmd(c *cli.Context) error {
 	}
 
 	if c.IsSet("format") {
-		return t.Execute(os.Stdout, buildah.GetBuildInfo(builder))
+		err := t.Execute(os.Stdout, buildah.GetBuildInfo(builder))
```

Comment:
Does the error case look OK without adding a println?

Reply:
you mean without adding extra info to the returned error?

Reply:
Ooops, sorry, too many reviews at once.  I was thinking the error would be printed here and might not have a carriage return as it should before the next shell prompt.  However the error handling code handles that and no issue there.  nvm me.